modules/grafana/files/dashboards/rummager_queues.json

```diff
@@ -75,22 +75,22 @@
           "grid": {
             "leftMin": 0
           },
-          "leftYAxisLabel": "message count"
+          "leftYAxisLabel": "Messages per minute"
         }
       ]
     },
     {
-      "height": "400px",
+      "height": "200px",
       "editable": true,
       "collapse": false,
       "panels": [
         {
-          "title": "Publishing queue consumer queue size (RabbitMQ)",
+          "title": "Publishing queue depth (RabbitMQ)",
```

Comment:
I think "queue size" is a more descriptive name understood by most developers.

Also, apparently RabbitMQ defines queue depth as:

> Sum of ready and unacknowledged messages (queue depth).

https://www.rabbitmq.com/man/rabbitmqctl.1.man.html